Lib/dbm/sqlite3.py

```diff
@@ -0,0 +1,117 @@
+import os
+import sqlite3
+from contextlib import suppress, closing
+from collections.abc import MutableMapping
+
+BUILD_TABLE = """
+  CREATE TABLE IF NOT EXISTS Dict (
+    key TEXT NOT NULL,
+    value BLOB NOT NULL,
+    PRIMARY KEY (key)
+  )
+"""
+GET_SIZE = "SELECT COUNT (key) FROM Dict"
+LOOKUP_KEY = "SELECT value FROM Dict WHERE key = ? LIMIT 1"
+STORE_KV = "REPLACE INTO Dict (key, value) VALUES (?, ?)"
+DELETE_KEY = "DELETE FROM Dict WHERE key = ?"
+ITER_KEYS = "SELECT key FROM Dict"
+
+
+class error(OSError):
+    pass
+
+
+def _normalize_uri_path(path):
+    path = path.replace("?", "%3f")
+    path = path.replace("#", "%23")
+    while "//" in path:
+        path = path.replace("//", "/")
+    return path
+
+
+class _Database(MutableMapping):
+
+    def __init__(self, path, flag):
+        match flag:
+            case "r":
+                flag = "ro"
+            case "w":
+                flag = "rw"
+            case "c":
+                flag = "rwc"
+            case "n":
+                flag = "rwc"
+                try:
+                    os.remove(path)
+                except FileNotFoundError:
+                    pass
+            case _:
+                raise ValueError("Flag must be one of 'r', 'w', 'c', or 'n', not",
+                                 repr(flag))
```

Comment:
Should there be a formatting operation?

Reply:
Yes, of course, this is not `print()` :)